compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -170,11 +198,17 @@ static bool isFullyDynamicOp(linalg::LinalgOp op) {
 }
 
 /// Returns the vectorization pre-processing strategy (peeling, masking) for the
-/// given LinalgOp, depending on the op traits and the target architecture.
+/// given LinalgOp. It is based on either:
+///   * user-specified value, or
+///   * heuristics (e.g. the op traits and the target architecture).
 static VectorPreProcStrategy
 getVectorPreProcStrategy(linalg::LinalgOp linalgOp) {
-  // Generic strategies.
+  // If set, use the strategy selected by a user.
+  if (clPProcStrategy != VectorPreProcStrategy::Heuristics) {
```

Comment:
Would it make sense to call this `Undefined`? Heuristics make sense to initialize this flag but it doesn't have a meaning within the rest of the code that uses these values. If we found `Heuristics` somewhere else it would mean that the strategy hasn't been defined yet.

Reply:
https://github.com/openxla/iree/pull/16165